viewer/src/components/SpaceHex.vue

```diff
@@ -94,15 +97,21 @@ export default class SpaceHex extends Vue {
     return this.$store.state.gaiaViewer.data.map;
   }
 
+  warning(hex: GaiaHex): string {
+    const warnings = this.highlightedHexes.get(hex)?.warnings;
+    return warnings != null && warnings.length > 0 ? warnings?.map(w => buildWarnings[w].text).join(", ") : null;
```

Comment:
```suggestion
    return warnings?.length > 0 ? warnings?.map(w => buildWarnings[w].text).join(", ") : null;
```

If `warnings` is null,  `warnings.length` is `undefined`, and any comparison with a number will return false.